modules/_nixos-disintegration/initrd.nix

```diff
@@ -5,8 +5,8 @@ let
 in
 {
   disabledModules = [
-    <nixpkgs/nixos/modules/tasks/encrypted-devices.nix>
-    <nixpkgs/nixos/modules/tasks/filesystems/zfs.nix>
+    "tasks/encrypted-devices.nix"
+    "tasks/filesystems/zfs.nix"
```

Comment:
Another place where we don't need the full nixpkgs path: https://nixos.org/manual/nixos/stable/#sec-replace-modules